src/index.ts

```diff
@@ -19,56 +19,51 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
  */
-import { CacheWorker } from './src/cache-worker'
-import { Service } from './src/lib'
-import { cacheKeys, pagination } from './config'
+import cacheKeys from "./cache-keys.json";
+import { CacheWorker } from "./cache-worker";
 
-start()
+start();
 
-async function start() {
+function start() {
   const cacheWorker = new CacheWorker({
     apiEndpoint: process.env.SERLO_ORG_HOST as string,
-    secret: process.env.SERLO_ORG_SECRET,
-    service: process.env.SERLO_SERVICE as Service,
-    pagination,
-  })
+    secret: process.env.SECRET,
+    service: process.env.SERVICE,
+    pagination: parseInt(process.env.PAGINATION as string),
```

Comment:
You can save yourself some headache by adding an `env.d.ts` file that describes all available env variables and their type, see e.g. https://github.com/serlo/api.serlo.org/blob/master/src/env.d.ts. This way,
- you don't need to work with `undefined` (and therefore don't need to typecast)
- have documented which env variables exist (so we easily know what to pass to the Docker image)